apps/web/src/pages/CertificationExamVouchersPage/CertificationExamVouchersPage.tsx

```diff
@@ -0,0 +1,348 @@
+                      "You've completed any preparatory work such as prerequisite training modules and are ready to take the certification exam",
+                    id: "oc4iCB",
+                    description:
+                      "Fourth item in list of eligibility requirements section",
+                  })}
+                </li>
+              </ul>
+            </p>
+          </div>
+          <div data-h2-margin-bottom="base(x3)">
+            <Heading
+              size="h3"
+              Icon={UserCircleIcon}
+              color="quaternary"
+              data-h2-margin-bottom="base(x1)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Apply and complete your profile",
+                id: "fxhdQd",
+                description:
+                  "Title for apply and complete your profile section",
+              })}
+            </Heading>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "To obtain a voucher, complete the application form. Your GC Digital Talent profile helps us confirm your eligibility. Before you apply, take a moment to review and update your information or create a profile if you haven't done so yet.",
+                id: "1uoeR6",
+                description:
+                  "First paragraph of apply and complete your profile section",
+              })}
+            </p>
+          </div>
+          <div data-h2-margin-bottom="base(x3)">
+            <Heading
+              size="h3"
+              Icon={QuestionMarkCircleIcon}
+              color="quinary"
+              data-h2-margin-bottom="base(x1)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "What to expect",
+                id: "8umYON",
+                description: "Title for what to expect section",
+              })}
+            </Heading>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "We'll respond to your application within 7 business days. If your request is approved, we'll send you a voucher code that you can use when booking your exam on the provider's platform.",
+                id: "9gU2in",
+                description: "First paragraph of what to expect section",
+              })}
+            </p>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "For questions regarding the vouchers, please contact <mailLink>icommunity-icollectivite@tbs-sct.gc.ca</mailLink>.",
+                  id: "I2kLuc",
+                  description: "Second paragraph of what to expect section",
+                },
+                { mailLink },
+              )}
+            </p>
+          </div>
+          <div data-h2-text-align="base(center)">
+            <Link
```

Comment:
According to Nienke's respone to your comment, this image should match the one in the hero.